Explicitly filter out Sanic warnings after Sanic is initialized (#1120)
RasaHQ/rasa-sdk: changelog/1120.misc.md

> Added the functionality that resets the Sanic warnings filter, which was allowing the triggering of Sanic warnings.

Reviewer: Reset Sanic warnings filter to prevent the triggering of Sanic warnings.

> Added the functionality that resets the Sanic warnings filter, which was allowing the triggering of Sanic warnings.

Reviewer: ```suggestion
Reset Sanic warnings filter to prevent the triggering of Sanic warnings.
```
